### DISCUSSION TITLE
ReferenceError: Property 'describe' doesn't exist, js engine: hermes

### DISCUSSION THREAD

**Discussion by kyusungpark** [expo/expo#24547] [Archived: Questions]
Hey everyone, 

I been adding some tests to my project recently and have been getting this error:
`ReferenceError: Property 'describe' doesn't exist, js engine: hermes`
<img width="344" alt="Screenshot 2023-09-20 at 4 46 19 PM" src="https://github.com/expo/expo/assets/58890841/114bb106-2d53-4823-81f6-f000094aa4af">

everytime I start my app. When I do run the tests, they run all fine, it seems to be just a little annoying error for now. 
I like to get rid of this since it shows up every single time I start the app.

Here are my configs and setting may help.
<img width="414" alt="Screenshot 2023-09-20 at 4 47 25 PM" src="https://github.com/expo/expo/assets/58890841/d2e57001-a57a-4b0c-82da-8d4cc587fa4a">
<img width="632" alt="Screenshot 2023-09-20 at 4 47 03 PM" src="https://github.com/expo/expo/assets/58890841/740ddfb6-2650-4ea6-85b5-d504a8f32cac">
<img width="248" alt="Screenshot 2023-09-20 at 4 46 35 PM" src="https://github.com/expo/expo/assets/58890841/9de8603f-74e1-48e2-bfd2-47fa02912f15">

**Comment by markitovtr1** [comment]
Just in case someone else faces this problem: in your scenario, this is caused probably because your tests folder is incorrectly named: has parenthesis and is missing `s`. Expo metro config adds `__tests__` as blockList.

If you are using another method to organize your tests, configure your Metro config file. I like to have a file with `.spec.ts` suffix for my tests, so I had to do this:

metro.config.js
```js
const { getDefaultConfig } = require('expo/metro-config')

const config = getDefaultConfig(__dirname)

config.resolver.blockList = [/(.*.spec.ts?)$/]

module.exports = config
```

Check [expo documentation](https://docs.expo.dev/guides/customizing-metro/) for more info on customizing metro bundler

**Reply by CodeNKoffee** [reply]
Thanks my G! Saved me a lot of time!

**Reply by hirwaaldo1** [reply]
Thank you
